ParabolInc/parabol
# chore(prompt-to-join): Allow to open add teammate dialog from the snackbar

**How to test:**

- Follow testing steps from #8086
- But now make sure you can open popup both from notification and snackbar
